versions/3.1.1.md

```diff
@@ -314,12 +314,24 @@ This is the root object of the [OpenAPI document](#openapi-description).
-| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. The list of values includes alternative security requirement objects that can be used. Only one of the security requirement objects need to be satisfied to authorize a request. Individual operations can override this definition. To make security optional, an empty security requirement (`{}`) can be included in the array.                                                                                                                                               |
+| <a name="oas-security"></a>security                    |  [[Security Requirement Object](#security-requirement-object)]  | A declaration of which security mechanisms can be used across the API. See [The `security` Field](#the-security-field).                                                                                                                                                                                                                                                                                                                                                                                                                                |
 | <a name="oas-tags"></a>tags                            |                  [[Tag Object](#tag-object)]                   | A list of tags used by the document with additional metadata. The order of the tags can be used to reflect on their order by the parsing tools. Not all tags that are used by the [Operation Object](#operation-object) must be declared. The tags that are not declared MAY be organized randomly or based on the tools' logic. Each tag name in the list MUST be unique.                                                                                                                                                                               |
 | <a name="oas-external-docs"></a>externalDocs            | [External Documentation Object](#external-documentation-object) | Additional external documentation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 
 This object MAY be extended with [Specification Extensions](#specification-extensions).
 
+###### The `security` Field
+
+The `security` field describes how requests are authorized:
+
+- If omitted, then nothing can be inferred about the authorization requirements; the behaviour is implementation-defined.
```

Comment:
While the text could have been more clear, I think this is actually pretty well-defined. I don't recall anyone arguing that any tools treat lack of security as anything but "allow all". The only question was whether [] is the same as that default or not.

Reply:
Defined, no. Understood / assumed, yeah, I'll buy that.

Happy to change this to "allow all", but I'll wait for more opinions first.

Reply:
I agree with @handrews  and I am not comfortable with omission being considered imp-defined. IMHO, this is counter productive.   "Allow all" seems (at least for now) a more reasonable approach.

Reply:
https://github.com/OAI/OpenAPI-Specification/pull/4007#discussion_r1711137553

Sounds like there are still some pretty directly opposed viewpoints around.

Reply:
For omitting `security`, it should be straightforward to check what a few major implementations do.  I'm 99.999% sure it is allow-all.  No one seemed to suggest otherwise on the call, and it's pretty inconceivable that leaving out a field that not everyone even wants to use would make a description un-callable.  Some people just describe the HTTP basics.

We could also check what these implementations do with `security: []` although even if they all do the same thing I'd hesitate to use that to define the behavior.